Tasks/VsTestV2/task.loc.json

```diff
@@ -17,8 +17,8 @@
   "author": "Microsoft Corporation",
   "version": {
     "Major": 2,
-    "Minor": 161,
-    "Patch": 2
+    "Minor": 162,
```

Comment:
you seem to be down grading the version, it has to become 2.162.3

The task won't get updated nor will the package get published

Reply:
changed it to 2.162.1